modules/core/src/main/java/org/apache/ignite/internal/managers/collision/GridCollisionManager.java

```diff
@@ -130,4 +129,11 @@ public void onCollision(
             });
         }
     }
+
+    /**
+     * @return Collision SPI.
+     */
+    public CollisionSpi collisionSpi() {
```

Comment:
This is literally equal to `ctx.config().getCollisionSpi()`, right? Why do you need this method?

Reply:
You're right, I can do that.